prep-html-skeleton/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Skeleton Exercise </title>
+      <body>
+
+        <h1>I built a proper HTML skeleton!</h1>
+
+      </body>
+  </head>
```

Comment:
Hi Andrew, be sure to divide your page into two specific sections. The `<head>` should open and close be fore the `<body>` opens and closes. The visible content of the page should go in the `<body>`.

```html
<!DOCTYPE html>
<html>
  <head>
    <!-- meta info like <title></title> goes in here -->
  </head>
  <body>
    <!-- visible content like <h1></h1> and <p></p> goes in here -->
  </body>
</html>
```